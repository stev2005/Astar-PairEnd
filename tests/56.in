ATATATGACTAGTTGACACTGGCAAGTCTCGAAGTCGGCTCGC
15
CTC
TTGCCTCCGCGCGCCT
CACTCAATCCCATATGGATCGATTTCCAGA
TA
G
CTCTAGTTGTCATAGCCCAAGGACGTATATCG
GCGCAGTTTATAGACTCCGTCTCAT
C
CTAAATCACAA
ATAGTCGAGATTTCAGCCCT
CTA
AACAAGCTGTTACTGCTG
AGTCTA
TCTTGTTTAACCTAAG
GGGTCAACGTGCTTAAGTGGTGGTT
