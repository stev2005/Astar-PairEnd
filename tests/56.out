0
7
15
0
0
13
11
0
4
8
0
7
1
7
10
